### ISSUE TITLE
It doesn't show what my typo is.

### ISSUE THREAD

**Issue by luther9** [kantord/LibreLingo#1192]
**Describe the bug**
In an exercise, I typed "buenos días, yo soy cecilia". The response was "You have a typo!" and "Correct spelling: Buenos días, yo soy Cecilia." There was no indication of what the typo was. LibreLingo usually ignores periods, but apparently not for this particular exercise. If I add a period, it's correct. If I capitalize the words without a period, it's a typo.

**To Reproduce**
1. Click on the "Phrases" lesson.
2. When you get to the voice exercise for "Buenos días, yo soy Cecilia.", type "buenos días, yo soy cecilia".

**Expected behavior**
The "Correct spelling" should clearly show what caused the typo flag.

**Screenshots**
![Screenshot_2021-04-02 LibreLingo](https://user-images.githubusercontent.com/1281696/113418824-3a732680-9394-11eb-8101-fc52b9cf917a.png)

**Desktop (please complete the following information):**
 - OS: PureOS amber
 - Browser Firefox
 - Version 78.8.0esr (64-bit)

**Comment by kantord** [comment]
Hi @luther9 thanks for the idea!

@allcontributors please add @luther9 for ideas

**Comment by kantord** [comment]
This is not trivial to do, but not very hard either.

Basically we need to add this logic to[ answer-corrector](https://github.com/kantord/LibreLingo/blob/main/apps/answer-corrector/src/index.ts#L71) and perhaps come up with a frontend design for it

**Comment by luther9** [comment]
I just looked at the [code for the exercises](https://github.com/kantord/LibreLingo/tree/main/courses/spanish-from-english). This is the only exercise out of the whole Spanish course that has an explicit period. That explains why all the other phrases ignore periods.

I think for consistency, either all the exercises should have periods, or none of them should.

When writing this bug report, I assumed there was something special about this sentence that tripped up something in the algorithm, which I can now see was wrong. The solution I suggested in my bug report might still be useful for hard-to-see typos or for catching bugs in the course YAML.

**Comment by kantord** [comment]
> I think for consistency, either all the exercises should have periods, or none of them should.

I would say none of them should, and it should be somehow enforced

**Comment by killuaDev** [comment]
I'm looking at doing this one, should the answer-corrector still be used? It looks like Listening challenge implements its own answer corrector.

**Comment by kantord** [comment]
yeah, answer corrector should be used, and I think it should be used in all cases :thinking:
